lms/product/moodle/_plugin/course_copy.py

```diff
@@ -0,0 +1,26 @@
+from lms.product.plugin.course_copy import CourseCopyFilesHelper, CourseCopyGroupsHelper
+from lms.services.moodle import MoodleAPIClient
+
+
+class MoodleCourseCopyPlugin:
```

Comment:
This plugin just calls the "course copy helpers". There's some duplication going on here but can't be fixed until we tackle some the other refactors I've been mentioning.